components/channel_header/channel_header.test.tsx

```diff
@@ -46,19 +43,19 @@ describe('components/ChannelHeader', () => {
 
     const populatedProps = {
         ...baseProps,
-        channel: {
+        channel: TestHelper.getChannelMock({
             id: 'channel_id',
             team_id: 'team_id',
             name: 'Test',
             delete_at: 0,
-        },
-        channelMember: {
+        }),
+        channelMember: TestHelper.getChannelMembershipMock({
             channel_id: 'channel_id',
             user_id: 'user_id',
-        },
+        }),
         currentUser: {
             id: 'user_id',
-        },
+        } as UserProfile,
```

Comment:
I couldn't use `TestHelper` here. When I do, `should render bot description` test is failing.

Reply:
That's interesting. I wonder why that happens...

I'm good with leaving it this way though

Reply:
After a second round of look, `currentUser` was missing `bot_description`. Now it is added and works fine